### PR TITLE
Added docs for the Gleap (Actions) destination

### PR DESCRIPTION
**Pull request by boehlerlukas**
### Proposed changes

This PR adds the Gleap (Actions) docs as specified in https://segment.com/docs/partners/destinations/#write-documentation.
Private Gleap (Actions) url: https://app.segment.com/gleap/destinations/catalog/gleap-cloud-actions

### Merge timing
- ASAP once approved

### CODE REVIEW

File: src/connections/destinations/catalog/gleap/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+---
+title: Gleap (Actions) Destination
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[Gleap](https://gleap.io/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} is a customer feedback platform designed for apps and websites. It offers a suite of tools including visual bug reporting, live chat, AI customer support, public roadmaps, marketing automation, and more, aimed at enhancing customer success and product improvement.
+
+> (delete after reading) Update your company name and support email address.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/gleap/index.html:10]:
```suggestion

```

File: src/connections/destinations/catalog/gleap/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+This destination is maintained by Gleap. For any issues with the destination, [contact their Support team](mailto:hello@gleap.io).
+
+> (delete after reading) The section below explains how to enable and configure the destination. Include any configuration steps not captured below. For example, obtaining an API key from your platform and any configuration steps required to connect to the destination.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/gleap/index.html:14]:
```suggestion

```

File: src/connections/destinations/catalog/gleap/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Getting started
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure Gleap**.
+4. Select an existing Source to connect to Gleap (Actions).
+5. To use the Gleap destination, obtain an API key by signing up for at [app.gleap.io](https://app.gleap.io).
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/gleap/index.html:22]:
```suggestion
5. To use the Gleap destination, obtain an API key by signing up at [app.gleap.io](https://app.gleap.io){:target="_blank”}.
```

File: src/connections/destinations/catalog/gleap/index.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+6. Once registered, navigate to Project > Settings > Security in the Gleap Dashboard.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/gleap/index.html:23]:
```suggestion
6. Once registered, navigate to **Project > Settings > Security** in the Gleap dashboard.
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/gleap/index.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
+
+---
+title: Gleap (Actions) Destination
+---
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/gleap/index.html:4]:
```suggestion
---
title: Gleap (Actions) Destination
id: 656f2474a919b7e6e4900265
beta: true
hidden: true
---
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]

**Comment by boehlerlukas** [comment]
@tcgilbert thanks a lot for the quick review and requested changes. I just committed them.

Once this PR is through, what are the next steps to get into public beta? We are having a few customers that would love to get their hands on the integration, but right now it seems we will need to privately invite them, so this would be greatly appriciated (If we are not missing something).

Thanks a lot for your help in moving this forward!

**Comment by tcgilbert** [comment]
> @tcgilbert thanks a lot for the quick review and requested changes. I just committed them.
> 
> Once this PR is through, what are the next steps to get into public beta? We are having a few customers that would love to get their hands on the integration, but right now it seems we will need to privately invite them, so this would be greatly appriciated (If we are not missing something).
> 
> Thanks a lot for your help in moving this forward!

I checked your catalog metadata, and we have everything we need on our side to go into Public Beta once these docs are through. From here, the only thing we ask is that you have confirmed you have tested your integration and it is working as expected. Could you confirm that for me?

**Comment by boehlerlukas** [comment]
@tcgilbert awesome! I just updated the path from gleap to gleap-cloud-actions.

We did test the integration ourselves and have 2 customers using it (one of them for a few weeks now). It looks good from our end :)

Thanks a lot!

**Review by tcgilbert — APPROVED** [review]

**Comment by tcgilbert** [comment]
@boehlerlukas 

> We did test the integration ourselves and have 2 customers using it (one of them for a few weeks now). It looks good from our end :)

Great! In that case, I will move your integration into public beta tomorrow afternoon when the docs site rebuilds and these docs are live. I will follow up through email to confirm. Meanwhile, if you have any other questions or concerns, please let me know.

**Review by rchinn-segment — APPROVED** [review]

**Comment by boehlerlukas** [comment]
@tcgilbert Awesome, thank you so much for the fast merging!
Have a wonderful day :)

**Pull request merged**